spec/input_validator_spec.rb

```diff
@@ -23,17 +23,25 @@
     end
 
     describe "#valid_token?" do
-        it "should return false if token is not X or O" do
+        it "should return false if token is not a letter from A to Z" do
             
-            expect(@input_v.valid_token?("P")).to eq(false)
             expect(@input_v.valid_token?("3")).to eq(false)
-            expect(@input_v.valid_token?("M")).to eq(false)
         end
 
-        it "should return true if token is X or O" do
+        it "should return false if token is a blank character" do
+            
+            expect(@input_v.valid_token?(" ")).to eq(false)
+        end
```

Comment:
Is `nil` a valid token? 🤔

Reply:
Thank you. I added a test and modification for that.